src/arch/amdgpu/vega/gpu_decoder.cc

```diff
@@ -2115,7 +2115,7 @@ namespace VegaISA
         &Decoder::decode_invalid,
```

Comment:
Not relevant to this patch, but one of these days we should add a comment for each invalid one so that it's easy for us to ID which one is the "right" one to add